src/FixedDataTable.js

```diff
@@ -608,6 +608,7 @@ var FixedDataTable = createReactClass({
   },
 
   _onRef(div) {
+    this._touchHandler.setRoot(div);
```

Comment:
please check variable this._touchHandler is not null before call setRoot func. 
I have tested this and it throw error this._touchHandler is undefined.